src/components/checkbox/style.css

```diff
@@ -1,16 +1,38 @@
-:host {
-	display: block;
+* {
+	box-sizing: border-box;
 }
-:host[hidden] {
-	display: none;
+
+:host {
+	display: flex;
+	justify-content: center;
+	align-items: baseline;
+	border: 1px solid rgb(var(--smoothly-color-contrast));
+	border-radius: 2px;
+	width: fit-content;
+	height: max-content;
+	margin: 0.3rem 0;
+	line-height: 0;
+
 }
+
 smoothly-icon {
-	cursor: pointer;
+	display: flex;
+	justify-content: center;	
+	align-items: center;
+}
+
+.outline {
+	stroke: rgb(var(--smoothly-color-contrast));
+	opacity: 1;
+	transition: opacity 200ms ease-in-out;
 }
-smoothly-icon:nth-child(2) {
-	position: absolute;
-	transform: scaleX(0.6);
+
+.hidden {
+	opacity: 0;
 }
-:host[disabled] smoothly-icon {
+
+:host[disabled] > smoothly-icon {
```

Comment:
there might be some more rules needed here. i did not see any difference on a disabled checkbox without a check mark in it (compared to a non disabled one)

Reply:
changed